### PR TITLE
Clarify setup - prevent everybody deactivating before running notebooks

### PR DESCRIPTION
**Pull request by pwalczysko**
Just to clarify - naive user might deactivate the freshly created env immediately otherwise.

cc @jburel

### CODE REVIEW

**Comment by jburel** [comment]
Could you make a similar change in workshop notebook?

**Comment by pwalczysko** [comment]
> Could you make a similar change in workshop notebook?

fixed in https://github.com/ome/ABIS-GBI-imaging-course-07-2023/pull/3/commits/2d99d4bf51d68913c09d92f6a70d38385eed64f9

File: notebooks/workshop.ipynb [hunk 1]
```diff
@@ -73,7 +73,9 @@
     "\n",
     "    $ conda activate abis_gbi_course_cellpose_2023\n",
     "    \n",
-    "Before creating a new environment, remember to deactivate the current one:\n",
+    "Now run the scripts or notebooks as desired in this environment. Go to the additional instructions for notebooks below.\n",
+    "\n",
+    "(When finished working with one environment, before creating a new environment, remember to deactivate the current one:)\n",
```
Comment (jburel) [inline notebooks/workshop.ipynb]: bracket and ``:`` seems strange
I think we should remove the brackets

Comment (pwalczysko) [reply notebooks/workshop.ipynb]: fixed in https://github.com/ome/ABIS-GBI-imaging-course-07-2023/pull/3/commits/1e3434e451cd91857d806331402fd1f965e8c6a8

**Pull request merged**